content/library/advanced-features/app-testing/get-started.md

```diff
@@ -0,0 +1,267 @@
+---
+title: Get started with app testing
+slug: /library/advanced-features/app-testing/get-started
+---
+
+# Get started with app testing
+
+Streamlit's app testing framework is not tied to any particular testing tool, but we'll use `pytest` for our examples since it is one of the most common Python test frameworks. To try out the examples in this guide, be sure to install `pytest` into your Streamlit development environment before you begin:
+
+```bash
+pip install pytest
+```
+
+## A simple testing example with `pytest`
+
+This section explains how a simple test is structured and executed with `pytest`. For a more complete introduction to `pytest`, check out Real Python's guide to [Effective Python testing with pytest](https://realpython.com/pytest-python-testing/).
+
+### How `pytest` is structured
+
+`pytest` uses a naming convention for files and functions to conveniently execute tests. Name your test scripts of the form `test_<name>.py` or `<name>_test.py`. For example, `test_myapp.py` or `myapp_test.py`. Within your test scripts, each test is written as a funtion, also named to begin or end with `test`. For our examples, we will prefix all our test scripts and test functions with `test_`.
```

Comment:
Nit: `funtion` -> `function`

Reply:
✅ In the proofread edits I just commited.